full-service/src/json_rpc/json_rpc_request.rs

```diff
@@ -88,24 +89,27 @@ pub enum JsonCommandRequest {
         txo_id: String,
         output_values: Vec<String>,
         destination_subaddress_index: Option<String>,
-        fee: Option<String>,
+        fee_value: Option<String>,
+        fee_token_id: Option<String>,
         tombstone_block: Option<String>,
     },
     build_transaction {
         account_id: String,
-        addresses_and_values: Option<Vec<(String, String)>>,
+        addresses_and_amounts: Option<Vec<(String, Amount)>>,
         recipient_public_address: Option<String>,
-        value_pmob: Option<String>,
+        amount: Option<Amount>,
         input_txo_ids: Option<Vec<String>>,
-        fee: Option<String>,
+        fee_value: Option<String>,
+        fee_token_id: Option<String>,
         tombstone_block: Option<String>,
         max_spendable_value: Option<String>,
     },
     build_unsigned_transaction {
         account_id: String,
         recipient_public_address: Option<String>,
-        value_pmob: Option<String>,
-        fee: Option<String>,
+        amount: Option<Amount>,
+        fee_value: Option<String>,
+        fee_token_id: Option<String>,
```

Comment:
```suggestion
        // Multi-Token Support
        amount: Option<Amount>,
        // MOB-Only
        value_pmob: Option<String>,
        // Multi-Token Support
        fee_amount: Option<Amount>,
        // MOB-Only
        fee: Option<String>,
```

Reply:
fee_token_id and fee_value are not both required, as if only 1 is provided then the API will figure out the default for the other.

If fee_token_id is set to "0" and fee_value is omitted, it will use the default minimum if available.